.github/ISSUE_TEMPLATE/Standard-Feature-Issue-Template.md

```diff
@@ -22,31 +22,44 @@ so that < why do you need this (short description of the business value >.
 ## Notes
 < bulleted list of special notes or suggestions to the development team on the implementation of the changes. >  
 
-## Definition of Ready
+## Ready for Planning
 
 - [ ] The Issue maintains standard format for Issues and meets INVEST
-- [ ] Issue Defined and written appropriately
-- [ ] Acceptance Criteria Defined
-- [ ] User acceptance test cases defined
+- [ ] Story and Business Value complete
+- [ ] Initial Acceptance Criteria Defined
 - [ ] The person who will Approve story identified
-- [ ] Dependencies Identified
+
+## Ready for Ready to Work
+- [ ] Acceptance Criteria Refined
 - [ ] Issue size estimated by Dev Team
-- [ ] Proposed solution accepted
+- [ ] Dependencies Identified
 
-## Definition of Done
+## Ready for Review
+- [ ] Proposed solution defined and accepted
+- [ ] Code defined, implemented, and built
+- [ ] Unit test defined, implemented, and passed
```

Comment:
We've not required devs to write unit tests for everything up to this point. Just testing in general. It can be unit, integration or acceptance. Are we intentionally changing that?

Reply:
changed per our discussion

Reply:
Why don't we want to have unit tests written by this point?

Reply:
Because devs are still writing Acceptance tests. If we also require them to write unit and integration tests, it will be a significant increase in workload on the team.